frontend-ssr/styles/_animations.scss

```diff
@@ -21,3 +21,26 @@
     opacity: 0.5;
   }
 }
+
+@keyframes dummyNotificationAnimation {
```

Comment:
We should use the same `skeletonOpacity` animation for the Notification Skeleton. Let's keep it consistent across the app - less code to write.